.github/workflows/test_package.yml

```diff
@@ -44,38 +44,24 @@ jobs:
       # if: steps.cache.outputs.cache-hit != 'true'
       run: |
         python -m pip install --upgrade --user pip
-        pip install -r requirements.txt
-        pip install -r requirements_dev.txt
+        pip install setuptools tox
         python --version
         pip --version
         pip list
 
     - name: Run tests
       run: |
+        # python -m tox
         # tox --sitepackages
+        pip install .[dependencies]
         python setup.py build_ext --inplace
         python -c 'import challenge_scoring'
-        # coverage run --source challenge_scoring -m pytest challenge_scoring -o junit_family=xunit2 -v --doctest-modules --junitxml=junit/test-results-${{ runner.os }}-${{ matrix.python-version }}.xml
-
-    - name: Upload pytest test results
-      uses: actions/upload-artifact@master
-      with:
-        name: pytest-results-${{ runner.os }}-${{ matrix.python-version }}
-        path: junit/test-results-${{ runner.os }}-${{ matrix.python-version }}.xml
-      # Use always() to always run this step to publish test results when there are test failures
-      if: always()
 
-    - name: Package Setup
-    # - name: Run tests with tox
+    - name: Build and install package
       run: |
-        pip install build
+        pip install -e .
```

Comment:
`pip install build` successfully builds the package but names it as `UNKNOWN` for some reason. Investigating it is left for a separate PR.